Description: Update manual page to refer to django-admin instead of django-admin.py
 Update the manual page to speak of django-admin instead of
 django-admin.py as that's the name used by the Debian package.
 .
 This is a Debian specific patch.
Forwarded: not-needed
Author: Brett Parker <iDunno@sommitrealweird.co.uk>
Origin: vendor

--- a/docs/man/django-admin.1
+++ b/docs/man/django-admin.1
@@ -1,8 +1,8 @@
-.TH "django-admin.py" "1" "March 2008" "Django Project" ""
+.TH "django-admin" "1" "March 2008" "Django Project" ""
 .SH "NAME"
-django\-admin.py \- Utility script for the Django Web framework
+django\-admin \- Utility script for the Django Web framework
 .SH "SYNOPSIS"
-.B django\-admin.py
+.B django\-admin
 .I <action>
 .B [options]
 .sp
